### PR TITLE
MueLu/Ifpack2: Making BlockRelaxation+LinearPartitioner grab the FixedBlockSize by default

### PR DESCRIPTION
**Pull request by csiefer2**
TESTS: RHEL6

### CODE REVIEW

File: packages/muelu/test/unit_tests/MueLu_TestHelpers.hpp [hunk 1]
```diff
@@ -191,9 +191,9 @@ namespace MueLuTests {
         GO numGlobalElements; //global_size_t
         if (matrixType == "Laplace1D")
           numGlobalElements = nx;
-        else if (matrixType == "Laplace2D" || matrixType == "Star2D")
+        else if (matrixType == "Laplace2D" || matrixType == "Star2D" || matrixType == "Elasticity2D")
```
[the remark this answers is not in the document]
Comment (csiefer2) [reply packages/muelu/test/unit_tests/MueLu_TestHelpers.hpp:194]: I was wondering why that didn't work...

[the remark this answers is not in the document]
Comment (csiefer2) [reply packages/muelu/test/unit_tests/MueLu_TestHelpers.hpp]: Fixed it

File: packages/muelu/test/unit_tests/MueLu_TestHelpers.hpp [hunk 2]
```diff
@@ -191,9 +191,9 @@ namespace MueLuTests {
...
           numGlobalElements = nx*ny;
-        else if (matrixType == "Laplace3D" || matrixType == "Brick3D")
+        else if (matrixType == "Laplace3D" || matrixType == "Brick3D" || matrixType == "Elasticity3D")
```
[the remark this answers is not in the document]
Comment (csiefer2) [reply packages/muelu/test/unit_tests/MueLu_TestHelpers.hpp]: Fixed

**Comment by csiefer2** [comment]
@trilinos/framework Random ROL failure.  Adding AT: RETEST

**Comment by prwolfe** [comment]
> @trilinos/framework Random ROL failure. Adding AT: RETEST

Looks like openmpi was attempting to open a file in /tmp and failed somehow.  This is old enough that the directory has been cleaned out (we have back to 4/25 and this happened 4/23.) None of the logs I have access to show anything for that date.  I don't know that there is anything I can do at this point.

Paul

**Pull request merged**